WalletWasabi.Gui/Rpc/WasabiJsonRpcService.cs

```diff
@@ -39,6 +39,17 @@ public object[] GetUnspentCoinList()
 			}).ToArray();
 		}
 
+		[JsonRpcMethod("createwallet")]
+		public object CreateWallet(string walletName, string password)
+		{
+			var walletGenerator = new WalletGenerator(Global.WalletsDir, Global.Network);
+			walletGenerator.TermsAccepted = true;   // Global.WalletService is {};
```

Comment:
If it is accepted by default, is it needed on the business logic level?

Reply:
Yes. The terms and Conditions are business logic and no UI logic. Here this is set to true because we ask to accept the T&C every time a user creates a new wallet file instead of asking to accept them only once when you run wasabi for first time and save that acceptance in some place. It is a bit weird to me. Why don't we ask to accept the T&C when the user recovers a wallet?